test/integration/test_raw_expect_100.py

```diff
@@ -0,0 +1,151 @@
+######################################################################
+#
+# File: test/integration/test_raw_expect_100.py
+#
+# Copyright 2023 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+import http.client
+import io
+import os
+import random
+import secrets
+import time
+
+import pytest
+
+from b2sdk.b2http import B2Http
+from b2sdk.encryption.setting import EncryptionSetting
+from b2sdk.encryption.types import EncryptionAlgorithm, EncryptionMode
+from b2sdk.exception import InvalidAuthToken
+from b2sdk.raw_api import REALM_URLS, B2RawHTTPApi
+from b2sdk.utils import hex_sha1_of_stream
+
+from .fixtures import b2_auth_data  # noqa
+
+
+@pytest.fixture
+def expect_100_setup(b2_auth_data, monkeypatch):  # noqa
+    application_key_id, application_key = b2_auth_data
+    raw_api = B2RawHTTPApi(B2Http())
+    realm = os.environ.get('B2_TEST_ENVIRONMENT', 'production')
+    realm_url = REALM_URLS.get(realm, realm)
+    auth_dict = raw_api.authorize_account(realm_url, application_key_id, application_key)
+
+    account_id = auth_dict['accountId']
+    account_auth_token = auth_dict['authorizationToken']
+    api_url = auth_dict['apiUrl']
+
+    sse_b2_aes = EncryptionSetting(
+        mode=EncryptionMode.SSE_B2,
+        algorithm=EncryptionAlgorithm.AES256,
+    )
+
+    bucket_name = 'test-raw-api-%s-%d-%d' % (
+        account_id, int(time.time()), random.randint(1000, 9999)
+    )
+
+    bucket_dict = raw_api.create_bucket(
+        api_url,
+        account_auth_token,
+        account_id,
+        bucket_name,
+        'allPublic',
+        is_file_lock_enabled=True,
+    )
```

Comment:
please reuse the single bucket for all tests here - there is not reason not to, and it will reduce the overall API calls i.e. make tests faster